libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -356,6 +356,9 @@ userdata Parameter method set_and_save boolean float'skip_check
 userdata Parameter method configured boolean
 userdata Parameter method set_default boolean float'skip_check
 
+include AP_Scripting/AP_Scripting.h
+singleton AP_Scripting rename scripting
+singleton AP_Scripting method restart void
```

Comment:
maybe we should make it clear that its all scripts. 
```suggestion
singleton AP_Scripting method restart_all void
```

Reply:
done